tests/serializers/test_json.py

```diff
@@ -10,7 +11,7 @@
 from django.test import SimpleTestCase, TestCase, TransactionTestCase
 from django.utils.translation import override, ugettext_lazy
 
-from .models import Score
+from .models import Score, ScoreDecimal
```

Comment:
What I mean is that ScoreDecimal can be inside the test method instead of in models.py.  See https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/unit-tests/#isolating-model-registration